amundsen_application/static/js/components/DashboardPage/styles.scss

```diff
@@ -2,6 +2,12 @@
 
 .dashboard-page {
 
+  .header-section {
+    .flag {
+      margin: 0 4px 0 0;
```

Comment:
Why 4px vs `$spacer-1`? My understanding from the new design concepts Wiebke has been pushing is that our default spacing is 8px (followed by multiples of 8). So unless I'm missing something in Figma I believe this would be 8px as usual.

Reply:
In the mock it looks a little closer than 8px, and also combines with the `&nbps;` before it. I will change to 8px on both sides.